monetio/models/gocart.py

```diff
@@ -0,0 +1,78 @@
+""" GOCART File Reader """
```

Comment:
```suggestion
"""GOCART File Reader."""
```
Also, could you add a bit more info about GOCART (maybe a link or two) and contexts that this reader is applicable to? Is it only for GEFS-Aerosols or is it applicable to other GOCART situations?

Reply:
Still looking into this - I'll get back to you soon